html/lesson-6.html

```diff
@@ -9,13 +9,36 @@
+                    <symbol xmlns="http://www.w3.org/2000/svg" viewBox="0 0 205 40" id="tripHouse">
+                        </symbol>
+                </svg>
+                     <svg class="trip">
+                        <use xlink:href="#tripHouse"></use>
+                      </svg>
                 <div class="trip-house-container">
                 <a class="text" href="#">Stays</a>
                 <a class="text" href="#"> Attractions</a>
-                <a class="Night" href="#"><img class="Night" src="../images/Night.svg" alt="Night"></a>
-                <a class="Account" href="#"><img class="Account" src="../images/AccountCircle.svg" alt="AccountCircle"></a>
+                <a class="Night" href="#">
+                    <!--<img class="Night" src="../images/Night.svg" alt="Night">-->
+                    <svg class="Night"> <path d="M25.61 10.61V4.39h-6.22L15 0l-4.39 4.39H4.39v6.22L0 15l4.39 4.39v6.22h6.22L15 30l4.39-4.39h6.22v-6.22L30 15l-4.39-4.39zM15 22.958a7.924 7.924 0 01-3.316-.73c2.732-1.26 4.642-4.018 4.642-7.228s-1.91-5.968-4.642-7.228A7.924 7.924 0 0115 7.042c4.39 0 7.958 3.568 7.958 7.958 0 4.39-3.568 7.958-7.958 7.958z"</path>
+                        <symbol xmlns="http://www.w3.org/2000/svg" viewBox="0 0 30 30" id="Night"> </symbol>
+                    </svg>
+                    <svg class="Night">
+                        <use xlink:href="#Night"></use>
+                        </svg>
+                     </a>
+                <a class="Account" href="#">
+                    <!--<img class="Account" src="../images/AccountCircle.svg" alt="AccountCircle">-->
+                    <svg  class="Account"> <path d="M20 0C8.96 0 0 8.96 0 20s8.96 20 20 20 20-8.96 20-20S31.04 0 20 0zm0 6c3.32 0 6 2.68 6 6s-2.68 6-6 6-6-2.68-6-6 2.68-6 6-6zm0 28.4c-5 0-9.42-2.56-12-6.44.06-3.98 8-6.16 12-6.16 3.98 0 11.94 2.18 12 6.16-2.58 3.88-7 6.44-12 6.44z"></path>
+                        <symbol  xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40" id="AccountCircle"></symbol>
+                      </svg>
+                    <svg class="Account">
+                    <use xlink:href="#AccountCircle"></use>
+                  </svg>
```

Comment:
It will be better if you put svgs in a separate file and call them by src="NameOfFile.svg#nameOfSvg